UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -724,4 +724,9 @@ public List<TextColor> InteractionsStrings()
 
 		return interactions;
 	}
+
+	public void CallToggleActionClient(bool toggled)
+	{
+		
```

Comment:
why is it empty?

Reply:
Because I add new method to interface IAction and that requires to add that to all other objects that implements Action